src/docs/commands_keybindings.md

```diff
@@ -1,122 +1,144 @@
+
+**helloworld-contribution.ts**
 ```typescript
 @injectable()
-export class EditorKeybindingContribution implements KeybindingContribution {
-
-    constructor(
-        @inject(EditorKeybindingContext) protected readonly editorKeybindingContext: EditorKeybindingContext
-    ) { }
-
-    registerKeybindings(registry: KeybindingRegistry): void {
-        [
-            {
-                command: 'editor.close',
-                context: this.editorKeybindingContext,
-                keybinding: "alt+w"
-            },
-            {
-                command: 'editor.close.all',
-                context: this.editorKeybindingContext,
-                keybinding: "alt+shift+w"
-            }
-        ].forEach(binding => {
-            registry.registerKeybinding(binding);
-        });
-    }
+export class HelloworldMenuContribution implements MenuContribution {
+ 
+   registerMenus(menus: MenuModelRegistry): void {
+       menus.registerMenuAction(CommonMenus.EDIT_FIND, {
+           commandId: HelloworldCommand.id,
+           label: HelloworldCommand.label
+       });
+   }
 }
 ```
 
-The `commandId` must be a unique command that was registered beforehand, the `context` is a simple class that makes sure the command/keybinding combination is enabled for the certain conditions. In the case of the editor, it looks like this
+To make our `MenuContribution` accessible to Theia, we need to bind the custom `HelloWorldMenuContribution` to the respective contribution symbol `MenuContribution`. This is done in the `helloworld-frontend-module`, for more details see [Services and Contributions](https://theia-ide.org/docs/services_and_contributions/).
 
-**editor-keybinding.ts**
+**helloworld-contribution.ts**
 ```typescript
-@injectable()
-export class EditorKeybindingContext implements KeybindingContext {
-    constructor( @inject(EditorManager) protected readonly editorService: EditorManager) { }
-
-    id = 'editor.keybinding.context';
+export default new ContainerModule(bind => {
+   bind(MenuContribution).to(HelloworldMenuContribution);
+      ...
+});
 
-    isEnabled(arg?: Keybinding) {
-        return this.editorService && !!this.editorService.activeEditor;
-    }
-}
 ```
 
-The context also has a unique id, which can then be used to register several keybindings to commands within that context. The `isEnabled()` method is where the context is evaluated to true or false for given conditions. Note that the context is an optional parameter that, when not provided, defaults to the `NOOP_CONTEXT`. Keybindings registered using this context will always have an enabled context, thus making them work anywhere in the application.
+Please note that you also contribute menu items without a command which allows you to create custom top level menus and sub menus. To achive this, contribute a menu item without a command and then reference the id of this menu item as a `MenuPath` in other contributes. This will add menu items to your custom menu.
```

Comment:
```suggestion
Please note that you also contribute menu items without a command which allows you to create custom top level menus and sub menus. To achieve this, contribute a menu item without a command and then reference the id of this menu item as a `MenuPath` in other contributes. This will add menu items to your custom menu.
```